cosmo_tester/framework/cluster.py

```diff
@@ -327,7 +451,7 @@ def _upload_plugin_to_manager(self, manager, plugin_name):
             x.id for x in manager.client.plugins.list()]
         try:
             manager.client.plugins.upload(plugin_wagon[0])
-            self._cfy.plugins.list()
+            manager.client.plugins.list()
```

Comment:
`cfy` isn't backwards compatible so it breaks when trying to run on the 3.4.2 manager.
If it were practical I'd prefer that we didn't use `cfy` at all in these tests (even if it's decided that we will fix backwards comaptibility of cfy/rest-client). It's an extra variable which could be broken for other reasons.